core/src/main/scala/sttp/tapir/model/StatusCodeRange.scala

```diff
@@ -0,0 +1,10 @@
+package sttp.tapir.model
+
+sealed abstract class StatusCodeRange(val range: Int)
+object StatusCodeRange {
+  case object Informational extends StatusCodeRange(1)
+  case object Success extends StatusCodeRange(1)
```

Comment:
Shouldn't it be `StatusCodeRange(2)` here?

Reply:
of course 🤦 can you maybe create a PR with a fix?

Reply:
https://github.com/softwaremill/tapir/pull/1924 but I'm getting out of memory errors in CI

Reply:
yeah the dotty compiler sometimes does that, no idea why